### ISSUE TITLE
App hangs with project loading message displayed

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#1511]
I have two projects imported from Door43, one Ephesians and one Titus.
I have been switching back and forth between the projects and the tN and tW tools for each several times, sometimes doing checks, sometimes just changing project and/or tool. After about 10 or so switches, the loading screen does not go away when attempting to open the project/tool.
Here is the text from the console:
[log for 1511.txt](https://github.com/unfoldingWord-dev/translationCore/files/990180/log.for.1511.txt)

**Comment by cckozie** [comment]
Similar problem from issue #1477 
Import and load https://git.door43.org/qa99/ab_tit_text_reg with tW
The loading dialog stays up forever

**Comment by cckozie** [comment]
Similar issue from  #1433 
I imported this project from door43 during a previous test: https://git.door43.org/qa98/ar_mat_text_ulb

I selected that project from the local Projects list, then selected the tW tool.
The loading modal comes up (not indicating the tool or showing any progress bar), but after the project loads, the loading modal stays in the foreground and cannot be dismissed.
The same behavior occurs if I load the project from the Projects modal.
I tried to capture the console log, but developer tools won't open using the hotkey sequence.
